### DISCUSSION TITLE
Display data from `loaders` to `control` block

### DISCUSSION THREAD

**Discussion by johnmorenodev** [storybookjs/storybook#26105] [Help]
### Summary

I'm new to  storybook. I'm experimenting with the [loaders](https://storybook.js.org/docs/writing-stories/loaders) functionality by fetching data and passing it to the component. I notice that when I do this, the `control` block won't show the data that was fetched. Is there any way to do this?

**Comment by valentinpalkovic** [comment]
Hi @johnmorenodev 

Thank you for your great question!

As you can see in our documentation about loaders, they quite differ from `args`. We are passing the `loaded` property into the Story callback function, which you can then attach or pass through to your component. Propagating the `loaded` value into `args` would be intuitive, but that's not how this feature was built. It would require a second `loaded` panel in controls to visualize and manually adjust the contents, which were fetched or calculated. This is, unfortunately, not implemented.
